lib/paper_trail/record_trail.rb

```diff
@@ -67,14 +67,25 @@ def previous_version
 
     def record_create
       return unless enabled?
-      event = Events::Create.new(@record, true)
+
+      build_version_on_create(in_after_callback: true).tap do |version|
+        version.save!
+        versions.reset
```

Comment:
Please add a comment explaining this `reset`.

Reply:
So, this `reset` is necessary because we are using `version_class.new` instead of `versions_assoc.build`? If so, can you mention this specifically in the comment please?

Reply:
Already done, looks like that:

```ruby
      build_version_on_create(in_after_callback: true).tap do |version|
        version.save!
        # One more Version object has been created. Need to invalidate
        # the `versions` association cache, otherwise the new version
        # can stay invisible within that association
        versions.reset
      end
```